packages/twenty-front/src/modules/activities/emails/components/EmailPreview.tsx

```diff
@@ -0,0 +1,88 @@
+import styled from '@emotion/styled';
+
+import { CardContent } from '@/ui/layout/card/components/CardContent';
+import { Avatar } from '@/users/components/Avatar';
+import { formatToHumanReadableDate } from '~/utils';
+
+import { Email } from '../types/email';
+
+const StyledContainer = styled(CardContent)`
```

Comment:
I feel StyledContainer is not a great naming here even if it's the parent container of EmailPreview. Maybe StyledCardContent to understand that this is actually a CardContent component

Reply:
Also CardContent, we might want to rename this one to CardContentContainer as it seems it actually is a container